tck/bridge-tck-main-portlet/src/main/java/com/liferay/faces/bridge/tck/tests/chapter_5/section_5_2/DependencyTrackingHeaderResponse.java

```diff
@@ -54,11 +46,11 @@ public DependencyTrackingHeaderResponse(HeaderResponse headerResponse) {
 	@Override
 	public void addDependency(String name, String scope, String version) {
 
-		for (String testHeadElementId : TEST_HEAD_ELEMENT_IDS) {
-
-			if (name.contains(testHeadElementId)) {
-				testHeadElementsAddedViaAddDependency.add(testHeadElementId);
-			}
+		// The portlet container may provide certain resources out-of-the-box. In that case, the bridge may elect not
+		// to provide the markup for the resource. In this test, jsf.js is the only resource that may potentially be
+		// provided in this manner.
+		if ("jsf.js".equals(name) && "javax.faces".equals(scope)) {
+			testHeadElementsAddedViaAddDependency.add("jsf.js");
```

Comment:
@stiemannkj1 Does `addDependency(String name, String scope, String version)` ever get called with Liferay Faces Bridge? Or is this just in there just in case a bridge extension does something different?